base/shell/explorer/explorer.cpp

```diff
@@ -164,6 +160,12 @@ StartWithDesktop(IN HINSTANCE hInstance)
     /* WinXP: Notify msgina to hide the welcome screen */
     if (!SetShellReadyEvent(L"msgina: ShellReadyEvent"))
         SetShellReadyEvent(L"Global\\msgina: ShellReadyEvent");
+
+    if (DoStartStartupItems(Tray))
```

Comment:
What's the motivation for moving this here instead of leaving it above?

Reply:
According to my API Monitor investigation, I think Windows Explorer does check its file title (explorer.exe) in its starting up.

Reply:
If not in 'shell ready' status, then opening a folder causes bad status (see the above doubled icons).